app/src/main/res/values/themes.xml

```diff
@@ -2,6 +2,7 @@
 <resources>
 
     <style name="Theme.Jerboa" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:windowTranslucentStatus">true</item>
```

Comment:
If i don't do this, it applies the status bar padding twice, this could have also been due to the nested scaffold.